com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/FrameSettingsUI.Drawers.cs

```diff
@@ -137,161 +142,96 @@ static HDRenderPipelineAsset GetHDRPAssetFor(Editor owner)
+
+            area.AmmendInfo(FrameSettingsField.LODBias,
+                overridedDefaultValue: hdrpAsset ? qualityLevelSettings.lodBias[serialized.lodBiasQualityLevel.intValue] : 0,
+                customGetter: () => serialized.lodBias.floatValue,
+                customSetter: v => serialized.lodBias.floatValue = (float)v,
+                overrideable: () => serialized.lodBiasMode.GetEnumValue<LODBiasMode>() != LODBiasMode.FromQualitySettings,
+                ignoreDependencies: true,
+                labelOverride: serialized.lodBiasMode.GetEnumValue<LODBiasMode>() == LODBiasMode.ScaleQualitySettings ? "Scale Factor" : "LOD Bias",
+                hasMixedValues: serialized.lodBias.hasMultipleDifferentValues);
+
+            area.AmmendInfo(
+                FrameSettingsField.MaximumLODLevelMode,
+                overridedDefaultValue: MaximumLODLevelMode.FromQualitySettings,
+                customGetter: () => serialized.maximumLODLevelMode.GetEnumValue<MaximumLODLevelMode>(),
+                customSetter: v => serialized.maximumLODLevelMode.SetEnumValue((MaximumLODLevelMode)v),
+                hasMixedValues: serialized.maximumLODLevelMode.hasMultipleDifferentValues
+            );
+            area.AmmendInfo(FrameSettingsField.MaximumLODLevelQualityLevel,
+                overridedDefaultValue: ScalableLevel3ForFrameSettingsUIOnly.Low,
+                customGetter: () => (ScalableLevel3ForFrameSettingsUIOnly)serialized.maximumLODLevelQualityLevel.intValue,
+                customSetter: v => serialized.maximumLODLevelQualityLevel.intValue = (int)v,
+                overrideable: () => serialized.maximumLODLevelMode.GetEnumValue<MaximumLODLevelMode>() != MaximumLODLevelMode.OverrideQualitySettings,
+                ignoreDependencies: true,
+                hasMixedValues: serialized.maximumLODLevelQualityLevel.hasMultipleDifferentValues);
+
+            area.AmmendInfo(FrameSettingsField.MaximumLODLevel,
+                overridedDefaultValue: hdrpAsset ? qualityLevelSettings.maximumLODLevel[serialized.maximumLODLevelQualityLevel.intValue] : 0,
+                customGetter: () => serialized.maximumLODLevel.intValue,
+                customSetter: v => serialized.maximumLODLevel.intValue = (int)v,
+                overrideable: () => serialized.maximumLODLevelMode.GetEnumValue<MaximumLODLevelMode>() != MaximumLODLevelMode.FromQualitySettings,
+                ignoreDependencies: true,
+                labelOverride: serialized.maximumLODLevelMode.GetEnumValue<MaximumLODLevelMode>() == MaximumLODLevelMode.OffsetQualitySettings ? "Offset Factor" : "Maximum LOD Level",
+                hasMixedValues: serialized.maximumLODLevel.hasMultipleDifferentValues);
+
+            area.AmmendInfo(FrameSettingsField.MaterialQualityLevel,
+                overridedDefaultValue: defaultFrameSettings?.materialQuality.Into() ?? MaterialQualityMode.Medium,
```

Comment:
Why it should be Medium above all other possibility? I mean if defaultFrameSettings is null, it means we are displaying the version in the global settings. So why it should be linked to Medium quality mode? This seams strange. At least it need comments to explain why we chose this default here.

Reply:
we needed to chose a value, i didnt know which one to use so i chose medium

Reply:
I think this need to be checked, perhaps ask TA and UX for this point.

Reply:
But it can be fixed in a later PR perhaps